config/peerpods/podvm/aws-podvm-image-cm.yaml

```diff
@@ -33,3 +33,8 @@ data:
   # NVIDIA GPU vars
   NVIDIA_DRIVER_VERSION: "535"
   NVIDIA_USERSPACE_VERSION: "1.13.5-1"
+
+  # Pause image vars
+  PAUSE_IMAGE_REPO: "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256"
+  PAUSE_IMAGE_VERSION: "ac97982e845413b2b7377171d1b4c85d2b2cfda174e8251e3049d4843cea40e1"
+  PAUSE_IMAGE_REPO_AUTH_FILE: "/tmp/regauth/auth.json"
```

Comment:
Wouldn't it be preferred to hide these from the user, i.e. set default values withing the scripts and comment these variables out?
Alternatively, can we simply use by default the current pause image used normally by Openshift?

Reply:
I agree.. If there is way to query the default pause for a release then we can set it in the operator, like we do for some other vars. Otherwise we can set it to a default (like above).

Reply:
@snir911 I couldn't find a way to query this info. One option is to mount the `/etc/crio/crio.conf.d/00-default` and read it. But it'll increase the required permissions for the container so I prefer to avoid it. 
For now, my suggestion is to use this approach of setting the default in configmap. Since this is a generic pause image it should work with other OCP versions as well. Having these options in the configmap will make it easier to replace it - using smaller size pause image, different one for a specific OCP release etc

Reply:
you can fetch it from the worker node mc, however, this indeed may be an overkill, i think it would be better to set it to some fixed value in the scripts (if not explicitly set) rather than having it in the cm. 
This will reduce confusion of users when customize cm image is required.

Reply:
oh, downside will be it will require to explicitly set for different architectures...

Reply:
So as you suggested, I'll set the default per arch (based on current info) in the script for now and leave it there..
Wdyt ?